ports/libsimpleservo/capi/Cargo.toml

```diff
@@ -17,6 +17,10 @@ simpleservo = { path = "../api" }
 log = "0.4"
 env_logger = "0.6"
 
+[target.'cfg(target_os = "windows")'.dependencies]
+libc = "0.2"
+winapi = {version = "0.3.7", features = ["winnt", "winbase", "processenv", "namedpipeapi", "ntdef", "minwindef", "handleapi", "debugapi"]}
```

Comment:
Could you change thus to "0.3" instead? I think we're using a different version here than the rest of Servo and we end up rebuilding everything.

Reply:
Will work on it ASAP.

Reply:
Ok, so it's not that. We should duplicate this list of features to https://github.com/servo/servo/blob/8f7440f36881fa60f4237d0dec8928799a6aa747/ports/glutin/Cargo.toml#L77 (keeping the existing features) and add windgi and winuser to this list. That will unify the features enabled across all builds, which sucks but will improve the build situation.

Reply:
Ok to leave version at "0.3.7" and add the 2 features?

Reply:
I think we need to add the two features to the libsimpleservo/capi/Cargo.toml, and add all the other features to glutin/Cargo.toml.

Reply:
Sounds good.  Added the additional features to glutin/Cargo.toml.  Building to verify.